src/data/ACTIONS/root/RDM.ts

```diff
@@ -34,37 +34,37 @@ export const RDM = ensureActions({
 		name: 'Contre Sixte',
 		icon: 'https://xivapi.com/i/003000/003217.png',
 		cooldown: 35000,
-		potency: 300, // Note 2nd enemy and others takes 50% less
+		potency: 360,
 	},
 	DISPLACEMENT: {
 		id: 7515,
 		name: 'Displacement',
 		icon: 'https://xivapi.com/i/003000/003211.png',
 		cooldown: 35000,
-		potency: 200,
+		potency: 180,
 		cooldownGroup: 10,
 	},
 	ENGAGEMENT: {
 		id: 16527,
 		name: 'Engagement',
 		icon: 'https://xivapi.com/i/003000/003231.png',
 		cooldown: 35000,
-		potency: 150,
+		potency: 180,
 		cooldownGroup: 10,
 	},
 	CORPS_A_CORPS: {
 		id: 7506,
 		name: 'Corps-a-corps',
 		icon: 'https://xivapi.com/i/003000/003204.png',
-		cooldown: 40000,
+		cooldown: 35000,
 		potency: 130,
```

Comment:
Need to add charges for the movement skills.

Reply:
Good catch.